packages/components/color-picker/src/use-color-slider.ts

```diff
@@ -354,6 +373,7 @@ export const useColorSlider = ({
         "aria-valuemax": max,
         "aria-valuemin": min,
         "aria-valuenow": value,
+        "aria-valuetext": ariaValueText,
```

Comment:
With `AlphaSlider`, the content of this `aria-valuetext` is not appropriate.

The `useColorSlider` must be used to switch the content depending on the destination.

Reply:
channel and modified it to display each appropriately.